readme.md

```diff
@@ -84,21 +93,37 @@ The following are valid Decimals:
 let a = 1.0;
 let b = 0.5;
 let c = 100.25;
+let d = 0.0;
+```
+
+The following are vaild negative Numbers:
+```
+let a = -1;
+let b = -0.5;
+let c = -0;
```

Comment:
Should -0 be valid? That feels kinda scary

Reply:
It just evaluates to 0, so I don't see why not. It could be useful when making a variable negative, but directly typed it does allow for unnecessary parts of programs.